test/functional/apps/discover/_runtime_fields_editor.ts

```diff
@@ -61,32 +60,37 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('allows creation of a new field', async function () {
-      await createRuntimeField('runtimefield');
+      const field = '_runtimefield';
```

Comment:
In case you may wonder, why starting with underscore? this way the field is sorted at the beginning of the sidebar and the doc viewer, which makes it much better to view what's going on in the screens if there's a failure ... also watching the test run gives benefits from that

Reply:
is this really enough to cause the test to stop failing? 😱

Reply:
no, added a note what solved the flakiness. When I resolved the flakiness, there was a permanent error. So we moved to a paginated list of fields ... and fields starting with `runtime...` were not on the first page -> error ... so renaming resolved this issue, and if there are follow up error screens we see the fields in the screenshots.